.github/workflows/job-precommit-java-mqtt-io.yml

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This workflow will run MQTT IO Unit Tests in Java SDK
+
+name: Java MQTT IO
```

Comment:
Should these workflows be removed from Jenkins?

Reply:
These workflows are part of the sharding of the Java Precommit workflow, some of the modules will be placed in a new java precommit workflow in https://github.com/apache/beam/pull/23138, while some others will remain in the current Jenkins job, mainly due to the lack of support for some reports.